### ISSUE TITLE
four loss functions called nan at the same time

### ISSUE THREAD

**Issue by ITEliteCCY** [junyanz/pytorch-CycleGAN-and-pix2pix#1004]
Hello, I'm using the pix2pix network to train CT images with 3D convolution.
Why are several loss functions called nan at the same time?In the first epoch that was it.
Nan was earlier in the first training, and it took a little longer to change the super parameters of Adam, but nan still appeared. I'm now training to the third epoch, which is still nan.

**Comment by junyanz** [comment]
If you use 3D Conv for the generator, do you also use the 3D Conv in the discriminator? As you change the network architectures, data, and applications, the original hyper-parameter might not work for you. I recommend that you first disable GAN loss and only use L1 loss, and make sure your 3D generator can produce reasonable results first.
